### PR TITLE
Automate choose account

### PR DESCRIPTION
**Pull request by Bur0k**
Based on #62 (Merge it before)

This solves fully or partially the following issues:
- #25 
- #27 (I am not sure about this)
- #40

Importing will always use the default time period.

### CODE REVIEW

**Comment by Bur0k** [comment]
Are there any concerns regarding this PR or was there not enough time to review/test it?

**Comment by bnw** [comment]
Hi, 
sorry for the delay. 
Could you rebase/cherry-pick the MR to the changes introduced in #62 so that "Files changed" only shows what actually changes in this MR? 
Thanks :)

**Comment by Bur0k** [comment]
I knew that i forgot something - Thanks for the heads up about the rebase :)

Added another commit so it does not log an error when the automation configs for these steps are missing. (Forgot that in my last PR)

File: app/public/html/choose-account.twig [hunk 1]
```diff
@@ -3,43 +3,77 @@
...
+            <button type="submit" class="btn btn-primary">Collect statements from your bank</button>
+        </form>
+
+        <script>
+            window.onload = function() {
+                setTimeout(function() {
+                    document.forms["next-form"].submit();
+                }, 2000);
+            }
+        </script>
```
Comment (bnw) [inline app/public/html/choose-account.twig:77]: Hm, this is a bit awkward:  
This leaves the user with 2 seconds to change the dates.  

How about adding the from and to date (e.g. in the form 'now - 1 month') to the config too? Then, we auto-skip this form iff all four values (bank_account, firefly_account, date_from, date_to) are provided.

**Review by bnw — CHANGES_REQUESTED** [review]

**Comment by Bur0k** [comment]
I moved the configs for this "Choose Automation" step into its own JSON Object - It became too cluttered for me.

Regarding your `from_date` and `to_date` comment: Is this a good way to implement it? I just passed it right into the `DateTime` constructor? This should not be a security issue, right?

**Review by bnw — APPROVED** [review]

**Comment by bnw** [comment]
Thanks a lot for your work and sorry for the delay :)

**Pull request merged**